dlinject.py

```diff
@@ -123,7 +151,7 @@ def dlinject(pid, lib_path, stopmethod="sigstop"):
 			log("Waiting for process to freeze...")
 			time.sleep(0.1)
 	else:
-		log.warn("We're not going to stop the process first!")
+		print("We're not going to stop the process first!")
```

Comment:
note to self, this should probably be `log()` instead of print but it's nbd (`log.warn` was definitely a bug though, I guess a leftover from when I was using pwntools)